src/utils/logger.js

```diff
@@ -1,13 +1,33 @@
-const noop = () => {};
+/* eslint-disable no-confusing-arrow */
+let debug = __DEV__;
 
-export const log = __DEV__ ? (...args) => console.log(
-  `%c ${args.join(' ')}`, 'background: #8cdcf2; padding: 3px 0; color: black;',
-) : noop;
+const prepareArgs = args => args.map(
+  value => typeof value === 'string' ? value : JSON.stringify(value),
+).join(' ');
 
-export const error = __DEV__ ? (...args) => console.log(
-  `%c ${args.join(' ')}`, 'background: red; padding: 3px 0; color: white;',
-) : noop;
+export const error = (...args) => {
```

Comment:
OK but if I return a noop by default, how could I change the export from the noop to the real logger function at runtime ? 😕